Code/Mantid/docs/source/interfaces/FilterEventUI.rst

```diff
@@ -0,0 +1,105 @@
+Filter Events Interface
+=======================
+
+.. contents:: Table of Contents
+  :local:
+  
+  
+Description
+-----------
+ 
+The Filter Events interface provides aid to users in order to set up event filter with different `Functionalities <http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#functionalities>`_. 
```

Comment:
That page is in the Wiki rather than the offline documentation, really that page should just be moved to the concepts documentation.